packages/vscode/README.md

```diff
@@ -24,8 +24,21 @@ Presentation <b>slide</b>s for <b>dev</b>elopers<br>
 - Re-ordering slides
 - Folding for slide blocks
 - Multiple slides project support
+- Start dev server in one click
 
-<img width="974" src="https://user-images.githubusercontent.com/11247099/116809994-cc2caa00-ab73-11eb-879f-60585747c3c9.png">
```

Comment:
I can't figure out how to generate URL like this. I was always getting `https://private-user-images.githubusercontent.com/...`. Not sure if the new format is fine.